### PR TITLE
rename arcmap TBX, and launch tools with separate BAT files

### PR DESCRIPTION
**Pull request by mfunk**
Updates for better file distribution and easier test runs on different clients.

## Description
Basically renamed the ArcMap toolbox from "Military_Tools_dsk.tbx" to "Military_Tools_arcmap.tbx", and separated test runner into two BAT files, and updated instructions.

## Related Issue
N/A

## Motivation and Context
Clarity of toolbox names and easier to run tests on client platforms.

## How Has This Been Tested?
Ran unit tests against new files.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by mfunk** [comment]
@ACueva 
Please change Jenkins to run the BAT files accordingly:

- For **ArcGIS Pro** run *.\utils\test\ArcGISProTestKickStart.bat*.
- For **ArcMap** run *.\utils\test\ArcMapTestKickStart.bat*

File: utils/test/ArcGISProTestKickStart.bat [hunk 1]
```diff
@@ -13,7 +13,7 @@ rem  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 rem  See the License for the specific language governing permissions and
 rem  limitations under the License.
 rem ------------------------------------------------------------------------------
-rem  TestKickStart.bat
+rem  ArcGISProTestKickStart.bat
```
Comment (ACueva) [inline utils/test/ArcGISProTestKickStart.bat]: I'll create an issue here to add both tests. https://github.com/ArcGIS/military-tools-for-arcgis-build

**Comment by ACueva** [comment]
Merged, kicking off Jenkins Build.

**Comment by ACueva** [comment]
@mfunk I added the calls for the unit tests.  ArcGIS Pro one is working. Kicked off another build to see if the ArcMap one is functioning after I made these changes.  The changes in the files are now showing up.  I think we're good for this pull request/issue.

**Pull request merged**